### DISCUSSION TITLE
build is taking a long time, isn't it ?

### DISCUSSION THREAD

**Discussion by setop** [mdn/yari#5510] [General]
Hi MDN folks,

I see the last [build](https://drop.zoocoop.com/d/egTMKgyP/L3PQhCA8dFZsjUjkUuadcVFFfKuM5czz) took 48 minutes to build the site at a speed of :

> Built 23,517 pages in 46.9 minutes, at a rate of 8.4 documents per second.

Compare to other static site generators - like for example hugo which can build 1'000 pages per second on a commodity laptop ; roughly 23 seconds for MDN content - it seems pretty slow.

Do you know what is causing this slowness ? Is it an issue for the contributors as the feedback loop must be very very long ?

Cheers

**Comment by caugner** [comment]
Hi @setop, good point! We are aware of this issue, and are planning on tackling it from April 2022. 🤞
